<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info_TableDIalog</class>
 <widget class="QDialog" name="Info_TableDIalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QTableWidget" name="info_table">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_clear">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::checked{
	background-color: rgb(85, 85, 255);
	color: rgb(255, 255, 255);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	border-color: rgb(28, 86, 15);
}</string>
        </property>
        <property name="text">
         <string>Clear Infomation</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_system_info">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::checked{
     background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.498, y2:0.989, stop:0 rgba(0, 170, 255, 185), stop:1 rgba(74, 207, 255, 180));
     border: 1px solid rgb(190, 190, 190);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
}</string>
        </property>
        <property name="text">
         <string>System Infomation</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_warning_info">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::checked{
     background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.498, y2:0.989, stop:0 rgba(0, 170, 255, 185), stop:1 rgba(74, 207, 255, 180));
     border: 1px solid rgb(190, 190, 190);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
}</string>
        </property>
        <property name="text">
         <string>Warning Infomation</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_alarm_info">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::checked{
     background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.498, y2:0.989, stop:0 rgba(0, 170, 255, 185), stop:1 rgba(74, 207, 255, 180));
     border: 1px solid rgb(190, 190, 190);
	 border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
}</string>
        </property>
        <property name="text">
         <string>Alarm Infomation</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
